src/Monolog/Handler/SlackHandler.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace MatchBot\Monolog\Handler;
+
+use Monolog\Handler\HandlerInterface;
+use Monolog\Logger;
+use Symfony\Component\Notifier\ChatterInterface;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+/**
+ * Sends error messages to our slack alarm channel so we deal with the errors without unnecessary delay.
+ *
+ * @phpstan-import-type Record from \Monolog\Logger
+ * @phpstan-import-type Level from \Monolog\Logger
+ * @psalm-import-type Record from \Monolog\Logger
+ * @psalm-import-type Level from \Monolog\Logger
+ *
+ */
+class SlackHandler implements HandlerInterface
```

Comment:
As discussed this class is probably unnecessary - may well delete on Monday and replace with https://github.com/Seldaek/monolog/blob/2.x/src/Monolog/Handler/SlackHandler.php